gnucash/gnome-utils/dialog-account.c

```diff
@@ -230,12 +230,13 @@ gnc_account_to_ui(AccountWindow *aw)
 
     string = xaccAccountGetColor (account);
 
-    if ((string == NULL) || (g_strcmp0 ("Not Set", string) == 0))
+    if (string == NULL)
         string = DEFAULT_COLOR;
-    if (gdk_rgba_parse(&color, string))
-    {
-        gtk_color_chooser_set_rgba(GTK_COLOR_CHOOSER(aw->color_entry_button), &color);
-    }
+
+    if (gdk_rgba_parse (&color, string) == FALSE)
```

Comment:
This is typically written as `if (!gdk_rgba_parse (&color, string))`

Reply:
I have a tendency to write the long version as it is clearer and then try to reduce then I have something working but missed this.